android/build.gradle

```diff
@@ -0,0 +1,63 @@
+plugins {
+    // TODO switch to com.android.library
+    //      (that will require exploration of the issue: by some
+    //      reason jUnit5 is not coming up for the tests,
+    //      so older jUnit is used)
+    id 'com.android.application'
+    id 'kotlin-android'
+    id "de.mannodermaus.android-junit5"  // jUnit5 for android
+}
+
+android {
+    compileSdk 31
+
+    defaultConfig {
+        minSdk 21
+        targetSdk 31
+
+        // TODO set that properly
+        versionCode 1
+        versionName "1.0"
+
+        // jUnit5 for android instrumented tests
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        testInstrumentationRunnerArgument "runnerBuilder", "de.mannodermaus.junit5.AndroidJUnit5Builder"
+    }
+
+    sourceSets {
+        androidTest {
+            kotlin.srcDirs += "../lib/src/test/kotlin"
+        }
+    }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
```

Comment:
very miner thing, can you add a blank space above?

Reply:
done